src/niswitch/metadata/functions.py

```diff
@@ -1205,6 +1216,7 @@
         'documentation': {
             'description': '\nConverts an error code returned by NI-SWITCH into a user-readable\nstring. Generally this information is supplied in error out of any\nNI-SWITCH VI. Use niSwitch_error_message for a static lookup of an\nerror code description.\n'
         },
+        'grpc_name': 'ErrorMessage',
```

Comment:
Although these came from our internal exports, we should avoid adding this unused information, if possible.

Reply:
We will use it later, just not in this PR.